index.html

```diff
@@ -88,49 +76,121 @@
 <script type="text/javascript" src="https://npmcdn.com/simple-statistics@2.0.0/dist/simple-statistics.min.js"></script>
 <script>
   var data = [];
-  var options = {
+    var options = {
     yearRange: 'all'
   };
+  var svg, x, y, xAxis, yAxis; 
+  var margin = { top: 15, right: 15, bottom: 30, left: 45 };
+  var width = 640 - margin.right - margin.left;
+  var height = 400 - margin.top - margin.bottom;
+  var xpadding = 35;
+  var ypadding = 10;
```

Comment:
Padding is generally an alternative to the margin pattern that we've demonstrated, and that you've used here. We'll see how this is leading to the axes being positioned weirdly.